admin/src/components/menu/SideMenu.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-navigation-drawer
+    expand-on-hover
+    rail
+    location='right'
+    permanent
+  >
+    <v-list>
+      <v-list-item
+        prepend-avatar="https://randomuser.me/api/portraits/women/85.jpg"
+        :subtitle="authStore.isLoggedIn ? 'Eingeloggt' : 'Nicht eingeloggt'"
+        :title="authStore.isLoggedIn ? 'Benutzer' : 'Gast'"
+      ></v-list-item>
+    </v-list>
```

Comment:
```suggestion
    <v-list>
      <v-list-item
        prepend-avatar="https://randomuser.me/api/portraits/women/85.jpg"
        :title="Erika Musterfrau"
      ></v-list-item>
    </v-list>
```

The entire sidebar should not get rendered if you're not authenticated. There is nothing that you can do with it otherwise.